From: Julien Viard de Galbert <julien@vdg.blogsite.org>
Date: Wed, 24 Nov 2010 00:16:20 +0100
Subject: Fix a spelling error

Thank you lintian ;)

Index: webalizer/wcmgr.c
===================================================================
--- webalizer.orig/wcmgr.c	2013-10-06 14:34:50.000000000 +0200
+++ webalizer/wcmgr.c	2013-10-06 14:45:59.000000000 +0200
@@ -692,7 +692,7 @@
    }
    dns_db->close(dns_db, 0);
    if (verbose)
-      printf("%s sucessfully deleted from cache file\n",addr);
+      printf("%s successfully deleted from cache file\n",addr);
 }
 
 /*********************************************/
@@ -770,7 +770,7 @@
          if (db_put(addr, name, (strcmp(name,addr))?0:1, runtime)==0)
             dns_db->close(dns_db,0);
          if (verbose)
-            printf("%s sucessfully added to cache file\n",addr);
+            printf("%s successfully added to cache file\n",addr);
       }
    }
 }
