--- webalizer-2.21-02/sample.conf~	2009-01-13 05:46:19.000000000 +0100
+++ webalizer-2.21-02/sample.conf	2010-09-08 11:59:02.556144035 +0200
@@ -26,7 +26,7 @@
 # (bzip2 compressed file), it will be decompressed on the fly as it
 # is being read.
 
-#LogFile        /var/lib/httpd/logs/access_log
+LogFile        /var/log/httpd/access_log
 
 # LogType defines the log type being processed.  Normally, the Webalizer
 # expects a CLF or Combined web server log as input.  Using this option,
@@ -107,10 +107,11 @@ LogFile        /var/log/httpd/access_log
 
 PageType	htm*
 PageType	cgi
-#PageType	phtml
+PageType	phtml
 PageType	php3
-#PageType	pl
+PageType	pl
 PageType	php
+PageType	shtml
 
 # PagePrefix allows all requests with a specified prefix to be
 # considered as 'pages'. If you want everything under /documents
@@ -546,13 +548,16 @@
 #HideReferrer	webalizer.org/
 
 # This one hides non-referrers ("-" Direct requests)
-#HideReferrer	Direct Request
+HideReferrer	Direct Request
+HideReferrer	file:*
 
 # Usually you want to hide these
 HideURL		*.gif
 HideURL		*.GIF
 HideURL		*.jpg
 HideURL		*.JPG
+HideURL		*.jpeg
+HideURL		*.JPEG
 HideURL		*.png
 HideURL		*.PNG
 HideURL		*.ra
@@ -571,10 +576,22 @@
 #GroupSite	*.aol.com
 #GroupSite	*.compuserve.com
 
-#GroupReferrer	yahoo.com/	Yahoo!
-#GroupReferrer	excite.com/     Excite
-#GroupReferrer	infoseek.com/   InfoSeek
-#GroupReferrer	webcrawler.com/ WebCrawler
+GroupReferrer	onet.pl/	Onet (onet.pl)
+GroupReferrer	wp.pl/		Wirtualna Polska (wp.pl)
+GroupReferrer	interia.pl/	Interia (interia.pl)
+GroupReferrer	.gazeta.pl/	Gazeta.pl (gazeta.pl)
+GroupReferrer	emulti.pl/	Emulti (emulti.pl)
+GroupReferrer	hoga.pl/	Hoga (hoga.pl)
+GroupReferrer	netsprint.	NetSprint (netsprint.pl)
+GroupReferrer	yandex.pl/	Yandex (yandex.pl)
+GroupReferrer	szukacz.pl/	Szukacz (szukacz.pl)
+GroupReferrer	yahoo.com/	Yahoo!
+GroupReferrer	altavista.com/	AltaVista (altavista.com)
+GroupReferrer	google.		Google (google.com)
+GroupReferrer	dmoz.org/	Open Directory Project (dmoz.org)
+GroupReferrer	excite.com/     Excite
+GroupReferrer	infoseek.com/   InfoSeek
+GroupReferrer	webcrawler.com/ WebCrawler
 
 #GroupUser      root            Admin users
 #GroupUser      admin           Admin users
@@ -584,29 +601,35 @@
 # for browsers, and not display all the detail records.
 # (You should use MangleAgent to refine further...)
 
-#GroupAgent	Opera/		Opera
+GroupAgent	Opera/		Opera
 #HideAgent	Opera/
-#GroupAgent	"MSIE 7"	Microsoft Internet Exploder 7
+GroupAgent	w3m*		w3m
+#HideAgent	w3m*
+GroupAgent	Mozilla		Netscape/Mozilla
+#HideAgnet	Mozilla
+GroupAgent	"MSIE 7"	Microsoft Internet Exploder 7
 #HideAgent	MSIE 7
-#GroupAgent	"MSIE 6"	Microsoft Internet Exploder 6
+GroupAgent	"MSIE 6"	Microsoft Internet Exploder 6
 #HideAgent	MSIE 6
-#GroupAgent	"MSIE "		Older Microsoft Exploders
+GroupAgent	"MSIE "		Older Microsoft Exploders
 #HideAgent	MSIE 
-#GroupAgent	Firefox/2.	Firefox 2
+GroupAgent	Firefox/2.	Firefox 2
 #HideAgent	Firefox/2.
-#GroupAgent	Firefox/1.	Firefox 1.x
+GroupAgent	Firefox/1.	Firefox 1.x
 #HideAgent	Firefox/1.
-#GroupAgent	Konqueror	Konqueror
+GroupAgent	Konqueror	Konqueror
 #HideAgent	Konqueror
-#GroupAgent	Safari		Safari
+GroupAgent	Safari		Safari
 #HideAgent	Safari
-#GroupAgent	Lynx*		Lynx
+GroupAgent	Lynx*		Lynx
 #HideAgent	Lynx*
-#GroupAgent	Wget/		WGet
+GroupAgent	Links*		Links
+#HideAgent	Links*
+GroupAgent	Wget/		WGet
 #HideAgent	Wget/
-#GroupAgent	(compatible;	Other Mozilla Compatibles
+GroupAgent	(compatible;	Other Mozilla Compatibles
 #HideAgent	(compatible;
-#GroupAgent	Mozilla*	Mozilla/Netscape
+GroupAgent	Mozilla*	Mozilla/Netscape
 #HideAgent	Mozilla*
 
 # HideAllSites allows forcing individual sites to be hidden in the

