--- ./sample.conf.org	Fri Sep 29 05:51:42 2000
+++ ./sample.conf	Tue Apr 13 21:30:06 2004
@@ -107,9 +107,11 @@
 
 PageType	htm*
 PageType	cgi
-#PageType	phtml
-#PageType	php3
-#PageType	pl
+PageType	phtml
+PageType	php3
+PageType	php
+PageType	pl
+PageType	shtml
 
 # UseHTTPS should be used if the analysis is being run on a
 # secure server, and links to urls should use 'https://' instead
@@ -383,7 +385,8 @@
 #HideReferrer	mrunix.net/
 
 # This one hides non-referrers ("-" Direct requests)
-#HideReferrer	Direct Request
+HideReferrer	Direct Request
+HideReferrer	file:*
 
 # Usually you want to hide these
 HideURL		*.gif
@@ -408,10 +411,22 @@
 #GroupSite	*.aol.com
 #GroupSite	*.compuserve.com
 
-#GroupReferrer	yahoo.com/	Yahoo!
-#GroupReferrer	excite.com/     Excite
-#GroupReferrer	infoseek.com/   InfoSeek
-#GroupReferrer	webcrawler.com/ WebCrawler
+GroupReferrer	onet.pl/	Onet (onet.pl)
+GroupReferrer	wp.pl/		Wirtualna Polska (wp.pl)
+GroupReferrer	interia.pl/	Interia (interia.pl)
+GroupReferrer	.gazeta.pl/	Gazeta.pl (gazeta.pl)
+GroupReferrer	emulti.pl/	Emulti (emulti.pl)
+GroupReferrer	hoga.pl/	Hoga (hoga.pl)
+GroupReferrer	netsprint.	NetSprint (netsprint.pl)
+GroupReferrer	yandex.pl/	Yandex (yandex.pl)
+GroupReferrer	szukacz.pl/	Szukacz (szukacz.pl)
+GroupReferrer	yahoo.com/	Yahoo! (yahoo.com)
+GroupReferrer	altavista.com/	AltaVista (altavista.com)
+GroupReferrer	google.		Google (google.com)
+GroupReferrer	dmoz.org/	Open Directory Project (dmoz.org)
+GroupReferrer	excite.com/     Excite
+GroupReferrer	infoseek.com/   InfoSeek
+GroupReferrer	webcrawler.com/ WebCrawler
 
 #GroupUser      root            Admin users
 #GroupUser      admin           Admin users
@@ -421,11 +436,16 @@
 # for browsers, and not display all the detail records.
 # (You should use MangleAgent to refine further...)
 
-#GroupAgent	MSIE		Micro$oft Internet Exploder
+GroupAgent	MSIE		Micro$oft Internet Exploder
+GroupAgent	Konqueror	Konqueror
+GroupAgent	Lynx*		Lynx
+GroupAgent	Links*		Links
+GroupAgent	Opera		Opera
+GroupAgent	w3m*		w3m
+GroupAgent	Mozilla		Netscape/Mozilla
 #HideAgent	MSIE
 #GroupAgent	Mozilla		Netscape
 #HideAgent	Mozilla
-#GroupAgent	Lynx*		Lynx
 #HideAgent	Lynx*
 
 # HideAllSites allows forcing individual sites to be hidden in the
@@ -516,9 +536,9 @@
 # engine, and the second is the URL variable used by that search engine
 # to define it's search terms.
 
-SearchEngine	yahoo.com	p=
-SearchEngine	altavista.com	q=
-SearchEngine	google.com	q=
+SearchEngine	yahoo.		p=
+SearchEngine	altavista.	q=
+SearchEngine	google.		q=
 SearchEngine	eureka.com	q=
 SearchEngine	lycos.com	query=
 SearchEngine	hotbot.com	MT=
