--- ./webalizer.c.org	Wed Apr 14 14:42:17 2004
+++ ./webalizer.c	Wed Apr 14 14:44:47 2004
@@ -231,9 +231,6 @@
 
 int main(int argc, char *argv[])
 {
-   setlocale (LC_ALL, "");
-   textdomain ("webalizer");
-   //   bindtextdomain ("webalizer", ".");
 
    int      i;                           /* generic counter             */
    char     *cp1, *cp2, *cp3, *str;      /* generic char pointers       */
@@ -262,6 +259,10 @@
                          "jul", "aug", "sep",
                          "oct", "nov", "dec"};
 
+   setlocale (LC_ALL, "");
+   textdomain ("webalizer");
+   //   bindtextdomain ("webalizer", ".");
+   
    /* initalize epoch */
    epoch=jdate(1,1,1970);                /* used for timestamp adj.     */
 
